jax/_src/extend/ffi.py

```diff
@@ -139,31 +147,36 @@ def _lowering(
     if "result_types" not in kwargs:
       kwargs["result_types"] = [mlir.aval_to_ir_type(aval) for aval in ctx.avals_out]
     if operand_layouts is None:
-      kwargs["operand_layouts"] = _default_layouts(aval.shape for aval in ctx.avals_in)  # pytype: disable=attribute-error
+      kwargs["operand_layouts"] = _default_layouts(ctx.avals_in)  # pytype: disable=attribute-error
```

Comment:
The ptype exception probably isn't needed anymore.

```suggestion
      kwargs["operand_layouts"] = _default_layouts(ctx.avals_in)
```